Task 6/Task6.2/REGISTRATION FORM.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Task 8.1</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+
+
+    <fieldset class="registration">
+
+        <h1>Регистрация</h1>
+        <hr>
+        <div class="block-registration">
+            <label for="email">E-mail</label><br>
+            <input type="email" id="email"><br>
+        </div>
```

Comment:
No need for `<br>` - you've already made input element wide enough to force them be on separate lines